### PR TITLE
Fixes #8959: Rudder-test cleanup

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/8959

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

File: scripts/cleanbox.sh [hunk 1]
```diff
@@ -184,14 +184,14 @@ fi
 # package that should exist everywhere
 ${PM_INSTALL} zsh vim less curl tree nano git binutils rsync
 # install that may fail
-${PM_INSTALL} htop
+${PM_INSTALL} htop uuidgen-runtime ldapscripts
```
Comment (fanf) [inline scripts/cleanbox.sh]: uuidgen-runtime should be added as a dependency in the packaging

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**